### ISSUE TITLE
A Logo for the group

### ISSUE THREAD

**Issue by kellertuer** [Julia-Users-Trondheim/julia-users-trondheim.github.io#1]
as @viniviena proposed, we should check for a nice Logo. Maybe its best to discuss ideas here and add it to the repository once we have found a nice logo.

**Comment by viniviena** [comment]
Hi :). 

I'll take a look at how other people are making logos for meetups and post here. I think this is going to bring some inspiration 😀

**Comment by kellertuer** [comment]
Hi @viniviena ,
sorry my December was a bit busier than I thought. Did you have time to think about a logo?

How shall we best proceed. I liked our format last time and think we should maybe first find a bit of a larger group before doing an introduction to Julia talk, what do you think? Maybe we could announce the next meeting on Innsida as well and try people at Sintef here in Trondheim (I do not know any one of them yet to be honest).

Maybe its best to establish a monthly meeting (every so-and-so-th weekday X per month) and do the larger talks at separate dates (maybe in the spots 2 weeks in between)?

**Comment by viniviena** [comment]
Hey Ronny, the end of the year was also hectic for me. I hope you had a great new year.

I looked into some logos from https://juliacon.org/2022/meetups/. It turns out that people just put a picture of a tourist place in the city with the Julia logo. We could do something similar, maybe. The question is, what is the most iconic place of Trondheim? Haha.

I agree that we need to bring more people before we do the Introduction to Julia. I think we need more people from NTNU and outside indeed. The point is how to attract those people. I know some people from SINTEF who work in Julia, and they would probably be glad to participate. Still, the general people at NTNU seem to have very little interest in programming - at least in chemical engineering, the interest is near zero. We could try to wrap up some specific application that brings attention and show how this is done in Julia and then put in some topics that people are less familiar with but that are more related to programming. 

How does it sound for you?

**Comment by kellertuer** [comment]
Then we could either take

* A view from Gamle Bybro
* An Image of Nidarosdomen – I would actually prefer some side view or the river side and not the front portal then
* the NTNU main building would maybe also a really nice idea to connect a bit to the university

I met some of my students from last semester today and they said they would be interested in an intro talk. So I might try to find time to prepare a few notebooks or see where I can “steal” some, probably from Guillaume Dalle

As I wrote, I would prefer to maybe to both, so people interested in exchanging User experience _and_ these talks. So if you have contact to some, let's try to get something of a meeting going :)

**Comment by viniviena** [comment]
Thanks for the great ideas for the logo. I will sketch something with the main building!

Regarding the talks/meetings, I agree that having both is better. I will talk with the friend I have at sintef about it. 

Since we are in mid January, do you think that having the meeting somewhere in the week of February 3rd - February 7th is a good time?

**Comment by kellertuer** [comment]
Looking forward to seeing the logo idea then :)

Yes for example. We could maybe fix a week for the meeting and one for talks? I do not have a preference when to do which.

So we could do

* every 2nd week of a month (Thursdays?) meeting
* every 4th week the spot for a talk if we have a talk

or something like that. For the meetings I can just get the room we had last time, which was nice to see whether someone was coming.
For the other – we can do that whenever needed.

**Comment by viniviena** [comment]
Hi Folks,

Here is a draft of the meetup logo. Sorry for the delay. It took me some time to find time to do it! 

Please give ideas and comments. 

![Image](https://github.com/user-attachments/assets/fcf3d595-cfe7-4b7b-b7c7-0096bcf14b98)

**Comment by kellertuer** [comment]
Nice!

No worries, my January was a bit busy so I also did not yet find time to work on the Julia intro talk, but maybe fixing a date somewhen in March is a good idea.

On my dark mode GitHub the logo looks a bit spooky, but in GIMP with a white BG – very nice!

I would maybe write “Trondheim, Norway” – and we could check for dark mode colors to make it look less spooky then, but besides these two minor remarks, this is a great logo!

**Comment by viniviena** [comment]
![Image](https://github.com/user-attachments/assets/02643699-87aa-4af0-84d0-92784db4a463)

Thanks for the feedback, Ronny. In dark BG, it looks weird, indeed. I've tried fitting a white BG in it.

Regarding Julia's intro, maybe 2nd or 3rd week of march would be a good target!

**Comment by viniviena** [comment]
Thanks for the feedback, Ronny. In dark BG, it looks weird, indeed. I've tried fitting a white BG in it.

Regarding Julia's intro, maybe 2nd or 3rd week of March would be a good target!

![Image](https://github.com/user-attachments/assets/2b1cd081-c3ff-43bf-80bd-c122568866cd)

**Comment by kellertuer** [comment]
Yes, we should then fix the date soon, so we have time to advertise it. For me, a Thursday would be great, so maybe March 20th? Maybe 17.00? 

We could also do a meeting like the last one two weeks before and check how to advertise that, e.g. mainly an experience exchange or so on March 6 that is?

**Comment by viniviena** [comment]
The 20th of March is a suitable date. That gives us plenty of time to announce and share at NTNU. 

About the time, at least here in the group, at 5 PM, at least 85% of the people are gone, and this number increases the closer it gets to Friday xD. Could we consider something earlier, like 15:45? It might be difficult to find a room at that time, but I think that it would increase the chance of people showing up.

**Comment by kellertuer** [comment]
That was exactly the trade-off I wanted to find, that we can find a reasonable room, but it is not yet too late. I can check for rooms at 16.00 later today, was thinking of maybe one of S2-S8 (in the central building).

**Comment by kellertuer** [comment]
I could get Sentralbygg 2, S5 for March 20, 2025, cf. https://i.ntnu.no/wiki/-/wiki/Norsk/S5+-+Undervisningsrom,
starting 16.00.

Maybe 16.00 is really better (0r we start 16.15) – since 14-16 is one lecture slot, so then students could just join from there.

**Comment by kellertuer** [comment]
So, then, how do we best announce the intro talk?

**Comment by kellertuer** [comment]
I just updated the homepage and added the logo,
let me know what you think!

For the announcement, we should maybe do a small poster we can hang in a few places and have an announcement text so send to a few students / lists on Innsida?

A small thing I noticed – both this repo, the URL and the former title was “Jluia User Group Trondheim”, your Logo now says “Meetup”...

**Comment by kellertuer** [comment]
How about the following (sorry for the monologue) – we use Julia User Group for the group (and the logo) and Meetup for the events?

I will try to find out how to announce stuff, but with all my conversation here, I feel I might probably then now be the only organiser. That is what I originally intended to avoid, that I do such an event completely by myself (preparing the talk, writing the announcement, booking the room, finding out how to announce, maybe do posters...)

**Comment by kellertuer** [comment]
Here is an announcement text (that I am not yet sure how to distribute)

```
[Workshop] Introduction to Julia
20/03/2025, 16.00, S5/SB2, NTNU, Trondheim.

[Julia](https://julialang.org) is a fairly new, open source programming language with scientific computing as one of its main targets. It allows for fast prototyping while also directly producing fast and high performance code, but also reproducible code.

In this workshop we first give an introduction to the language, how to install Julia, get started and work with [Pluto](https://plutojl.org) notebooks.
We will roughly follow the [Julia Modern Workflow](https://modernjuliaworkflows.org/writing/) project presentation, focussing a bit also on how to start programming in Julia if you come from Python, Matlab or R.
Afterwards we will help you get started and have a workshop to work on concrete ideas, firstsmall projects you want to get started with!

Join us on March 20th, 2024, 16.00, in S5 (Sentralbygg 2).
```

**Comment by viniviena** [comment]
Hi,

I would be more than glad to take responsibility for organisational aspects. I took it for the logo and don't see any problem doing other things. It is just a matter of agreeing.

We must set realistic expectations for each person's work capacity. The intent is not to overload anyone. 

 I can:

- Make an advertisement and place it in insidda. Send emails for mailing lists. Print it and put it in appropriate places.

- I will make the change for the Julia users group.


* * The announcement text is great. The same goes for the date and time. If you think it is useful, I can add some graphics to it.

**Comment by kellertuer** [comment]
Cool! I was already looking a bit into advertisements, but did not yet find much...
If you have an announcement I can post it to the IE faculty Innsida channel / the local IMF channel / our math students

If the text is good I can post that tomorrow or Monday – maybe a poster with the graphics would be great? Along Stripa there is a few places we could hang that.

For the Logo – I think it fits well on the homepage and we can update it when you fixed the text :)

**Comment by viniviena** [comment]
I can work on the graphics during the weekend, and we can announce it on Monday! I can also share it here at our department—we have a group on Teams. I can also check on how to announce in cybernetics.

**Comment by kellertuer** [comment]
Monday sounds good! I can definetly post it to our math students and the math department and hopefully find out how to post it to the IE faculty even

**Comment by viniviena** [comment]
![Image](https://github.com/user-attachments/assets/cd7126f6-83c2-48df-9fc9-fa152d2b25a7)

Updated logo!

**Comment by kellertuer** [comment]
Homepage also updated :)

**Comment by kellertuer** [comment]
I wanted to give you push rights here in the repo and then noticed – you already have that! So you can even change texts on the homepage as well if you like.

But great! So then if we have a poster and/or the text above, we can do the announcements on Monday.

**Comment by kellertuer** [comment]
Anything new on the announcements? I at least visited our optimisation lecture, where I announced the seminar and hope to start with the presentation preparation today as well.

**Comment by viniviena** [comment]
Still working on it. Should be finished by the end of the day :D.

**Comment by kellertuer** [comment]
I started the presentation with a first draft of the content in slide titles at 
https://github.com/Julia-Users-Trondheim/Intro-to-Julia

and can quickly hack a poster as well I think.

Let me know when I can maybe write an announcement to any other channel as well (unless you can already post to the IE factually channel).

**Comment by kellertuer** [comment]
My brain did not want to do so much math yet, so here is a first draft of a poster 

https://github.com/Julia-Users-Trondheim/Intro-to-Julia/blob/main/poster/intro-to-julia-2025.pdf

(QR code links to our homepage, the link below)

**Comment by viniviena** [comment]
![Image](https://github.com/user-attachments/assets/81c47f21-62dd-4e2e-8f79-dc5c39f0a7b8)


Is it good? Any suggestions? PS: I took your picture from GitHub, but if you prefer to don't show any picture or another one, let me know!

**Comment by kellertuer** [comment]
Nice approach for an announcement!
Minor stuff

* “PhD” on my side is wrong. I have a German Dr. so if it would be “Dr. Ronny Bergmann” (or with full decorations: Dr. rer. nat. habil.) – O do not care. even better: If it fits a single line, maybe just my name is enough. I am not my title.
* My email is underlined, yours is not, should maybe be unified?
* bullets on the left, maybe “Main features” -> “Get started” (or first steps)  and the last seems a bit long (and repetitive). Maybe “Workshop: Hands on with Julia” or so?
* itemises on the left: the last one repeats the two ones before, Maybe just “This Workshop is for you!” and a next one “Learn writing Julia code”?

At the very top the “Worhsop Program” sounds a bit too bold. Could we just leave that out? The left already says workshop?

**Comment by kellertuer** [comment]
Concerning my picture – I am also fine with an announcement without pictures. This is not about me (see the poster – does not mention my name) ;) But what ever you prefer. Maybe do not cut it out to white at least? This a-bit-of-a-background looks strange.

**Comment by viniviena** [comment]
![Image](https://github.com/user-attachments/assets/6ae04038-d192-4ae4-9942-c6c75fa8209f)

Thaks for the suggestions Ronny :D. I think that having a picture is good. I know it is not about you or any person, but people get more receptive about things (specially abstract things such as programming) when they see a picture of someone. If you don't have problem about having a picture, I'd keep it.

**Comment by kellertuer** [comment]
THat looks better. Why does the left have dat&time but the top line only the date?

Besides that the one thing that is different from my (more meant for the analogue world – already on my door) and your digital advertisement is, that I write “SB II” and you “Sentralbygg 2” ;) 
Minor note: Is there a space too much before “This workshop is for you”? To me it looks like that. Might also be just misleading in comparison to the Ws before.

For the image – I do not mind much – but with the background it looks a bit calmer.

**Comment by viniviena** [comment]
![Image](https://github.com/user-attachments/assets/2a69f6ee-f323-4404-bbec-044205adfbfb)


Thanks for feedback! Here is the most updated version :D

**Comment by kellertuer** [comment]
Aha! It was a space! 

Yes, this looks good, let's get that out into the world. Do you know how to announce this, maybe even to faculties like IE?
I think from IE I can repost to the Math department and maybe our students as well.

I have the poster (linked atop) on my door as well and will check how to maybe put a few on Stripa, since there are poster places.

**Comment by viniviena** [comment]
Last time we did a seminar with Chris, Idelfonso announced at my department mailing list (Chemical Engineering) and also in Innsida. I can check how he did it and possibly expand to other faculties. I was also thinking about printing some copies of it in a regular printer and sticking them in some doors (people do it all the time in K4).  I think it may be sufficient. I can also talk with my departments administration to share it in LinkedIn :).

**Comment by kellertuer** [comment]
I think for printing my poster is maybe nicer with its QR code and such

**Comment by viniviena** [comment]
Yep. I agree!

**Comment by kellertuer** [comment]
Perfect! And I do agree, that for the digital announcements, yours is definetly much better. I printed the poster in A3, that looks quite nice (I printed it 2 times since for the first I had to fix a typo9.

Nice, nice!

**Comment by kellertuer** [comment]
Published your image with the text from the poster below and a link to our homepage to
* IE faculty
* Department of Mathematical Sciences
* Matteland (where our master students hang out)
* pushed the image also to the other repository where I started the talk and where the print-poser-pdf is: https://github.com/Julia-Users-Trondheim/Intro-to-Julia

Room is reserved, so besides annoying more people with advertisements, there is mainly the talk to be created. Feel free to open issues with ideas on the other repo or fix typos (you have push rights on that repo).

edit: And posted to our thread on discourse.

**Comment by kellertuer** [comment]
Linkedin would be an idea, too? With my name on I feel a bit strange posting it, but I could repost it if you post it?

**Comment by viniviena** [comment]
Thanks Ronny :). I will also send it to people through mailing lists in our group/department. I will take a look at the presentation and add contributions

**Comment by kellertuer** [comment]
@viniviena if you have a bit of time, I finished most of the basics today, see

https://github.com/Julia-Users-Trondheim/Intro-to-Julia/blob/main/presentation/introduction-to-julia.pdf

MIssing:
* write a notebook containing all code from the presentation so I can easily copy that over during the presentation in when illustrating that I a notebook
* write a few details about pluto
* write a small Pluto demo that includes plotting

But everything until slide 23 should be finished. The remaining ones should already be there as empty ones.

**Comment by viniviena** [comment]
Hey Ronny. The slides look pretty good! Thanks for it. 

 I can support you on these points. By Thursday I can work them out. Does it work for you?

**Comment by kellertuer** [comment]
help either with commits or if you are unsure and want to discuss, open PRs on that repo.

The slides should be easy it is just 2 left and a bit depending on what I plan to talk about.

But the two Pluto notebooks would be great.
* the one with the examples from the slides – then I can copy that code over and do not always have to type it
* a Pluto demo with some (Makie) plotting and some interactivity – maybe a slider or a variable to change where one can see that everything updates

**Comment by kellertuer** [comment]
Now all slides should be complete; I also posted to BlueSky and LinkedIn today.

If you have time you could check the slides for typos. The main PDF now is also without transitions (that I use in the talk then).

**Comment by viniviena** [comment]
hey @kellertuer. I was checking the repo to work on the notebooks.

 I noticed a file named all_examples.jl though. Is it the first notebook you mentioned? Should I just work on the second notebook (Pluto demo)?

**Comment by kellertuer** [comment]
Yes I finished the examples, because my next 3 days are super busy and I am really relying on having these on the side Thursday. So I finished that on Saturday. Sorry for not writing that here, my weekend was also a bit busy.

Yes a demo would still be nice, something nice with plotting. Otherwise I will improvise something either the hour before the presentation (if time permits) or live.

**Comment by viniviena** [comment]
No worries. You have done a ton of things for Thursday - there is nothing to apologize for. I can do a demo with an interactive plot and push to the repo.

**Comment by kellertuer** [comment]
Ok. I also checked the room on Saturday already and that my laptop works fine there – so besides that I only have to twiddle a bit with my OBS settings so that the browser and Terminal show up nicely.

**Comment by viniviena** [comment]
Hey @kellertuer. I don't have push permission for the repo with the slides. I have made a pluto notebook with a classification problem example and the effect of regularization on the decision boundary.  I'm going to send by mail :).

**Comment by kellertuer** [comment]
In the long run we could ask cormulion (something like that he is written) to do the logo in a bit more graphics designer approach, but until then let's close this.
